src/astro/utils/sql_handler.py

```diff
@@ -0,0 +1,97 @@
+from typing import Dict, Optional
+
+from airflow.exceptions import AirflowException
+from airflow.operators.dummy import BaseOperator
+from sqlalchemy.sql.functions import Function
+
+from astro.databases.base import BaseDatabase
+from astro.settings import SCHEMA
+from astro.sql.table import Metadata, Table
+
+
+class SQLHandler(BaseOperator):
+    """Contains helper functions for creating and running SQL queries"""
```

Comment:
I think this class is doing many things, do you think it would be better if we have a class that
1. Resolves sql_queries and parameter templates
2. Then takes care of executing them with the Database package. 

and the 1. dropping of table and 2. Schema operation can be taken care of in Operator itself.